lib/registries/addon/routes.ts

```diff
@@ -3,6 +3,9 @@ import buildRoutes from 'ember-engines/routes';
 export default buildRoutes(function() {
     this.route('index', { path: '/registries' });
     this.route('discover', { path: '/registries/discover' });
+    this.route('branded', { path: '/registries/:providerId' } as any, function() {
```

Comment:
```suggestion
    this.route('branded', { path: '/registries/:providerId' }, function() {
```
this `as any` shouldn't be necessary, does it error if you remove it? ideally we'd avoid `any` to the extent possible

Reply:
I was just copying the pattern from the overview route. Will remove.

Reply:
oh huh, i wonder if the typedefs used to require some fudging... in any case, travis successfully picked up this request, so yay!